nginx/etc/nginx/nginx-ssl.conf.ctmpl

```diff
@@ -0,0 +1,95 @@
+user  nginx;
+worker_processes  1;
```

Comment:
> I would do the test in the Nginx reload.sh script and pass an environment variable to the consul-template call.

Makes sense.

It looks like there'd be two conditional blocks (and they could be collapsed into one), but that's from the person reading it, not implementing it ;-).